src/frontend/screens/Settings/ProjectSettings/YourTeam/index.tsx

```diff
@@ -173,10 +174,8 @@ export const YourTeam: NativeNavigationComponent<'YourTeam'> = ({
           name={participant.name || ''}
           deviceId={participant.deviceId}
           deviceType="mobile"
-          // This is a weak check. We should be using deviceIds, but those are not exposed
-          thisDevice={
-            deviceInfo.data && deviceInfo.data.name === participant.name
-          }
+          dateAdded={participant.joinedAt}
+          thisDevice={deviceInfo.data?.deviceId === participant.deviceId}
```

Comment:
this is semi out of scope, but small enough fix that I thought it wouldn't hurt to fix it here

Reply:
also out of scope, but ideally this would make sure that the `deviceInfo` query is in a success state before allowing this check to happen. Otherwise you can expose UI or features (or in a really unfortunate scenario, effects) unintentionally for a brief moment.

Think the work to properly account for that that can happen in an eventual follow-up though